docs/tutorials/migrations/nvm.md

```diff
@@ -0,0 +1,122 @@
+Run `flox init` from the relevant project directory (see [creating environments][creating_tutorial]{:target="\_blank"} for details), and then search for the version of Node.js that is currently included in your `.nvmrc`, or in the `"engines"` property of your `package.json`. (Note that node is listed as `nodejs` and variants thereof in the Flox catalog.)
+
+```zsh
+➜  node-project git:(main) ✗ flox search nodejs
+nodejs       Event-driven I/O framework for the V8 JavaScript engine
+nodejs_14    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_16    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_18    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_19    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_20    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_21    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_22    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_23    Event-driven I/O framework for the V8 JavaScript engine
+nodejs-10_x  Event-driven I/O framework for the V8 JavaScript engine
+
+Showing 10 of 67 results. Use `flox search nodejs --all` to see the full list.
+
+Use 'flox show <package>' to see available versions
+```
+
+There is _a lot_ of software in the Flox catalog, so you should have no trouble finding the Node.js version you need. For this project, perhaps you initially decide to install Node.js v20.18.1, "Iron" (LTS).
+
+```zsh
+➜  node-project git:(main) ✗ flox install nodejs
+✅ 'nodejs' installed to environment 'node-project'
+```
+## Verify the Node.js version
+Now that you've installed Node.js, you activate the Flox environment to verify that it has the the version you expect.
+
+```zsh
+➜  node-project git:(main) ✗ flox activate
+✅ You are now using the environment 'node-project'.
+To stop using this environment, type 'exit'
+
+...
+flox [node-project] ➜  node-project git:(main) ✗ node -v
+v20.18.1
+```
+
+## Update the Node.js version
+If you want to install a different node version, you can edit your environment to include the version you need.
+
+Perhaps you decide to upgrade node version to v22. To achieve this, you run `flox uninstall nodejs`, followed by `flox install nodejs_22`.
+
+```zsh
+➜  node-project git:(main) ✗ flox uninstall nodejs
+🗑️  'nodejs' uninstalled from environment 'node-project'
+➜  node-project git:(main) ✗ flox install nodejs_22
+✅ 'nodejs_22' installed to environment 'node-project'
+➜  node-project git:(main) ✗ flox list
+nodejs_22: nodejs_22 (nodejs-22.10.0)
+➜  node-project git:(main) ✗ flox activate
+✅ You are now using the environment 'node-project'.
+To stop using this environment, type 'exit'
+
+...
+flox [node-project] ➜  node-project git:(main) ✗ node -v
+v22.10.0
+```
+
+If you'd like more granular control over the Node.js version to install, you can get the correct Flox catalog version name for the desired Node.js version by running `flox show <package>`:
```

Comment:
I think this section belongs before the upgrade to v22, otherwise it seems like the upgrade process is "uninstall, then install the different version" as opposed to `flox upgrade`. Then you can frame the upgrade to v22 as "suppose you want a specific version".

Reply:
I was having a little bit of trouble with a straight upgrade here, and I think that's perhaps a quirk of how node versions end up in nixpkgs. I'll try a few things tomorrow to see if I can work out the kinks with the upgrade. The main issue is that node versions higher than 20 don't see to be included if you try to `flox install nodejs` as opposed to `flox install nodejs_22`. I wonder if that's still an "upgrade" if those are different packages in the catalog.